packages/components/src/components/form/BottomText.tsx

```diff
@@ -30,30 +30,63 @@ export const Container = styled.div<{ $inputState?: InputState; $isDisabled?: bo
         $isDisabled ? theme.textDisabled : getInputStateTextColor($inputState, theme)};
     ${typography.label}
     animation: ${slideDown} 0.18s ease-in-out forwards;
+    background-color: ${({ $inputState, theme }) =>
+        $inputState === 'info' && theme.backgroundAlertBlueSubtleOnElevation1};
+    border-radius: ${borders.radii.xxs};
+
+    ${({ $inputState, theme }) =>
+        $inputState === 'info' &&
+        `
+        margin: ${spacingsPx.sm} 0;
+        padding: ${spacingsPx.xs} ${spacingsPx.sm};
+        ::before {
+            content: '';
+            position: absolute;
+            top: -${spacingsPx.xs};
+            left: 20px;
+            transform: translateX(-50%);
+            width: 0;
+            height: 0;
+            border-left: 8px solid transparent; 
+            border-right: 8px solid transparent; 
+            border-bottom: 8px solid ${theme.backgroundAlertBlueSubtleOnElevation1}; 
+        }
+        `}
+`;
+
+export const Divider = styled.div`
+    height: inherit;
+    align-self: stretch;
+    width: 2px;
+    background-color: ${({ theme }) => theme.backgroundSurfaceElevation1};
+    margin: -${spacingsPx.xs} ${spacingsPx.xs};
 `;
 
 interface BottomTextProps {
     inputState?: InputState;
     isDisabled?: boolean;
-    icon?: IconName;
     children: ReactNode;
 }
 
-export const BottomText = ({
-    inputState,
-    isDisabled,
-    icon = 'warningCircle',
-    children,
-}: BottomTextProps) => {
+export const BottomText = ({ inputState, isDisabled, children }: BottomTextProps) => {
     const theme = useTheme();
+    const iconMap = {
+        error: 'warningCircle',
+        warning: 'warningCircle',
+        info: 'doubleCheck',
+    } as const satisfies Record<InputState, IconName>;
```

Comment:
We can use default icon for each input state, but we should be able to choose any icon with different prop.

Please take a look on component Warning (`packages/components/src/components/Warning/Warning.tsx`) and refactor it in the same style.

<img width="1064" alt="Screenshot 2024-06-07 at 14 24 11" src="https://github.com/trezor/trezor-suite/assets/858321/466c6dfa-f74a-41de-8e32-2bf67f751081">

<img width="1011" alt="Screenshot 2024-06-07 at 14 25 58" src="https://github.com/trezor/trezor-suite/assets/858321/7e2e0e1d-3a50-43f7-b963-12293c1f6262">

Reply:
This behavior is introduced now in BottomText component
`
 let iconName;

    if (icon) {
        iconName = icon;
    } else if (inputState && iconMap[inputState]) {
        iconName = iconMap[inputState];
    }

    return (
        <Container $inputState={inputState} $isDisabled={isDisabled}>
            {withIcon && iconName && <Icon name={iconName} size="medium" color={iconColor} />}
            
  `